convert2rhel/unit_tests/actions/pre_ponr_changes/backup_system_test.py

```diff
@@ -297,37 +304,45 @@ def test_backup_package_file_complete(
         backup_package_files_action,
         global_backup_control,
     ):
-        # Prepare the rpm -va ouput to the PRE_RPM_VA_LOG_FILENAME file
+        ### Prepare the 'rpm -Va' ouput to the PRE_RPM_VA_LOG_FILENAME file
```

Comment:
Since I'm a bit unclear about the triple hashes. What is the significance of `#` vs `###`?

Reply:
Probably just some typo or re-commenting the same comment two more times

Reply:
I thought so too but then saw a comment else where in the PR with

```
### Prepare the files from the 'rpm -Va' output by creating them
# Use only unique paths, since one path can be present multiple times in the output
# https://issues.redhat.com/browse/RHELC-1606
```

Reply:
The plan was to create some headings to each part of the test. But I think it wasn't good plan :D I'll remove that, since it's unclear

Reply:
No it's fine but was curious of the reason why. Wonder if we can make it more clear somehow..

I think when it's like this of a heading and comments underneath I'd go with this or something like it
```
# Heading
# 
# Text here
# Text here
```